### PR TITLE
ENH: latest lecture changes + QTradableStocks update

### PR DESCRIPTION
**Pull request by mmargenot**

### CODE REVIEW

**Comment by jmccorriston** [comment]
`from quantopian.pipeline import QTradableStocksUS` doesn't work for me in the IDE. Were you able to run all of the notebooks successfully?

**Comment by mmargenot** [comment]
Good catch, it's supposed to be `quantopian.pipeline.filters`

**Comment by ssanderson** [comment]
IMO we should put `QTradableStocksUS` on the top-level `quantopian.pipeline` module.  We do that already for a small number of highly-important classes.

**Comment by ssanderson** [comment]
the code that builds that namespace is here: https://github.com/quantopian/qexec/blob/master/qexec/quanto_namespaces.py#L310-L340

File: notebooks/lectures/Factor_Analysis/notebook.ipynb [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "from quantopian.pipeline.data.builtin import USEquityPricing\n",
     "from quantopian.pipeline.factors import CustomFactor, Returns\n",
     "from quantopian.pipeline.classifiers.fundamentals import Sector  \n",
-    "from quantopian.pipeline.experimental import QTradableStocksUS\n",
+    "from quantopian.pipeline import QTradableStocksUS\n",
```
Comment (ernestoeperez88) [inline notebooks/lectures/Factor_Analysis/notebook.ipynb]: I think this import is supposed to be:

```
from quantopian.pipeline.filters import QTradableStocksUS
```

File: notebooks/lectures/Factor_Analysis/preview.html [hunk 2]
```diff
@@ -11804,7 +11804,7 @@ <h1 id="Factor-Analysis">Factor Analysis<a class="anchor-link" href="#Factor-Ana
 <span class="kn">from</span> <span class="nn">quantopian.pipeline.data.builtin</span> <span class="kn">import</span> <span class="n">USEquityPricing</span>
 <span class="kn">from</span> <span class="nn">quantopian.pipeline.factors</span> <span class="kn">import</span> <span class="n">CustomFactor</span><span class="p">,</span> <span class="n">Returns</span>
 <span class="kn">from</span> <span class="nn">quantopian.pipeline.classifiers.fundamentals</span> <span class="kn">import</span> <span class="n">Sector</span>  
-<span class="kn">from</span> <span class="nn">quantopian.pipeline.experimental</span> <span class="kn">import</span> <span class="n">QTradableStocksUS</span>
+<span class="kn">from</span> <span class="nn">quantopian.pipeline</span> <span class="kn">import</span> <span class="n">QTradableStocksUS</span>
```
Comment (ernestoeperez88) [inline notebooks/lectures/Factor_Analysis/preview.html]: Same as above, QTradableStocksUS lives in quantopian.pipeline.filters

File: notebooks/lectures/Factor_Based_Risk_Management/notebook.ipynb [hunk 3]
```diff
@@ -92,7 +92,7 @@
     "from quantopian.pipeline.data import Fundamentals\n",
     "from quantopian.pipeline.data.builtin import USEquityPricing\n",
     "from quantopian.pipeline.factors import CustomFactor, Returns\n",
-    "from quantopian.pipeline.experimental import QTradableStocksUS\n",
+    "from quantopian.pipeline import QTradableStocksUS\n",
```
Comment (ernestoeperez88) [inline notebooks/lectures/Factor_Based_Risk_Management/notebook.ipynb]:
```
from quantopian.pipeline.filters import QTradableStocksUS
```

File: notebooks/lectures/Fundamental_Factor_Models/preview.html [hunk 4]
```diff
@@ -11826,7 +11826,7 @@ <h3 id="Portfolio-Construction-as-an-Asset-Pricing-Model">Portfolio Construction
 <span class="kn">from</span> <span class="nn">quantopian.pipeline.data</span> <span class="kn">import</span> <span class="n">Fundamentals</span>
 <span class="kn">from</span> <span class="nn">quantopian.pipeline.factors</span> <span class="kn">import</span> <span class="n">CustomFactor</span><span class="p">,</span> <span class="n">Returns</span><span class="p">,</span> <span class="n">Latest</span>
 <span class="kn">from</span> <span class="nn">quantopian.pipeline.classifiers</span> <span class="kn">import</span> <span class="n">Classifier</span>
-<span class="kn">from</span> <span class="nn">quantopian.pipeline.experimental</span> <span class="kn">import</span> <span class="n">QTradableStocksUS</span>
+<span class="kn">from</span> <span class="nn">quantopian.pipeline</span> <span class="kn">import</span> <span class="n">QTradableStocksUS</span>
```
Comment (ernestoeperez88) [inline notebooks/lectures/Fundamental_Factor_Models/preview.html]: QTradableStocksUS lives in quantopian.pipeline.filters

File: notebooks/lectures/Fundamental_Factor_Models/notebook.ipynb [hunk 5]
```diff
@@ -77,7 +77,7 @@
     "from quantopian.pipeline.data import Fundamentals\n",
     "from quantopian.pipeline.factors import CustomFactor, Returns, Latest\n",
     "from quantopian.pipeline.classifiers import Classifier\n",
-    "from quantopian.pipeline.experimental import QTradableStocksUS\n",
+    "from quantopian.pipeline import QTradableStocksUS\n",
```
Comment (ernestoeperez88) [inline notebooks/lectures/Fundamental_Factor_Models/notebook.ipynb]:
```
from quantopian.pipeline.filters import QTradableStocksUS
```

File: notebooks/lectures/Factor_Based_Risk_Management/preview.html [hunk 6]
```diff
@@ -11853,7 +11853,7 @@ <h3 id="Universe">Universe<a class="anchor-link" href="#Universe">&#182;</a></h3
 <span class="kn">from</span> <span class="nn">quantopian.pipeline.data</span> <span class="kn">import</span> <span class="n">Fundamentals</span>
 <span class="kn">from</span> <span class="nn">quantopian.pipeline.data.builtin</span> <span class="kn">import</span> <span class="n">USEquityPricing</span>
 <span class="kn">from</span> <span class="nn">quantopian.pipeline.factors</span> <span class="kn">import</span> <span class="n">CustomFactor</span><span class="p">,</span> <span class="n">Returns</span>
-<span class="kn">from</span> <span class="nn">quantopian.pipeline.experimental</span> <span class="kn">import</span> <span class="n">QTradableStocksUS</span>
+<span class="kn">from</span> <span class="nn">quantopian.pipeline</span> <span class="kn">import</span> <span class="n">QTradableStocksUS</span>
```
Comment (ernestoeperez88) [inline notebooks/lectures/Factor_Based_Risk_Management/preview.html]: QTradableStocksUS lives in quantopian.pipeline.filters

File: notebooks/tutorials/1_getting_started_lesson3/notebook.ipynb [hunk 7]
```diff
@@ -144,7 +144,7 @@
     "from quantopian.pipeline.factors import SimpleMovingAverage\n",
     "\n",
     "# Import built-in trading universe\n",
-    "from quantopian.pipeline.experimental import QTradableStocksUS\n",
+    "from quantopian.pipeline import QTradableStocksUS\n",
```
Comment (ernestoeperez88) [inline notebooks/tutorials/1_getting_started_lesson3/notebook.ipynb]:
```
from quantopian.pipeline import QTradableStocksUS
```

File: notebooks/tutorials/1_getting_started_lesson4/notebook.ipynb [hunk 8]
```diff
@@ -44,7 +44,7 @@
     "from quantopian.pipeline import Pipeline\n",
     "from quantopian.pipeline.data.psychsignal import stocktwits\n",
     "from quantopian.pipeline.factors import SimpleMovingAverage\n",
-    "from quantopian.pipeline.experimental import QTradableStocksUS\n",
+    "from quantopian.pipeline import QTradableStocksUS\n",
```
Comment (ernestoeperez88) [inline notebooks/tutorials/1_getting_started_lesson4/notebook.ipynb]:
```
from quantopian.pipeline.filters import QTradableStocksUS
```

File: notebooks/tutorials/1_getting_started_lesson4/notebook.ipynb [hunk 9]
```diff
@@ -84,7 +84,7 @@
...
```
Comment (ernestoeperez88) [inline notebooks/tutorials/1_getting_started_lesson4/notebook.ipynb]:
```
from quantopian.pipeline.filters import QTradableStocksUS
```

**Pull request merged**